index.html

```diff
@@ -5,11 +5,28 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Skillz</title>
+    <link rel="stylesheet" href="./style.css">
     <link rel="stylesheet" href="./styles/common.css">
     <link rel="stylesheet" href="./styles/normalize.css">
 </head>
 <body>
-    <h1>Skillz</h1>
+    <header>
+        <div class="leftBlock"> 
+            <img src="./img/header/logo.png">
+        </div>
+        <div class="centerBlock"> 
+            <ul class="navigation">
+                <li>Courses</li>
+                <li>Free Workshops</li>
+                <li>Blog</li>
+                <li>About</li>
+                <li>Contact</li>
```

Comment:
In items you need to have links with text.
We should to navigate by this links over our document

example:
```<li><a href='#courses'>Courses</a></li>```